storage-service/src/minio.ts

```diff
@@ -52,7 +53,7 @@ export const getMinioStatus = async () => {
 const bucketName: string = config.storage.bucketName;
 
 const makeBucket = (bucket: string, cb: Function) => {
-  minioClient.bucketExists(bucket, (err: any, exists: any) => {
+  minioClient.bucketExists(bucket, (err: unknown, exists: unknown) => {
```

Comment:
do not use `unknow` as type, exist is `boolean`, error is `Error`, more information [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/40f00cd698640f1d8077f4d52eed1b5d2b2a7838/types/minio/index.d.ts)